src/components/MovieList.js

```diff
@@ -0,0 +1,139 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const MovieList = ({ movies, setMovies }) => {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [runtimeMins, setRuntimeMins] = useState("");
+  const [url, setUrl] = useState("");
+  const token = localStorage.getItem("token");
+  const navigate = useNavigate();
+
+  const handleTitle = (e) => {
+    setTitle(e.target.value);
+  };
+  const handleDescription = (e) => {
+    setDescription(e.target.value);
+  };
+  const handleRuntime = (e) => {
+    if (e.target.value >= 0) {
+      setRuntimeMins(e.target.value);
+    }
+  };
+
+  const handleUrl = (e) => {
+    setUrl(e.target.value);
+  };
+  const handleLogout = () => {
+    navigate(`/`);
+    setLoggedIn(false);
```

Comment:
ok so you've even implemented logout, it seems you've made a fully functional webpage